ansible/playbooks/roles/kubernetes_master/tasks/master-init.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Load default variables for k8s nodes
```

Comment:
After moving parameter `enable_controller_attach_detach` to `schema/common/defaults/configuration/kubernetes-master.yml` loading default variables for k8s nodes will be obsolete.

Reply:
Done